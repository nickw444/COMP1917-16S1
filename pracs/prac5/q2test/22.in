22
4 -2 -9 -2 3 -10 7 3 4 9 6 -9 9 10 -2 2 -6 7 0 3 4 9
