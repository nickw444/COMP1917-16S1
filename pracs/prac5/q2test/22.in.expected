average of array: 1.818182
